static/locales/en-US.yaml

```diff
@@ -145,7 +146,17 @@ Settings:
     Clear Default Instance: Clear Default Instance
     View all Invidious instance information: View all Invidious instance information
     Region for Trending: Region for Trending
-        #! List countries
+    #! List countries
+    External Link Handling:
+      External Link Handling: External Link Handling
+      External Link Handling Tooltip: |
+        External Link Handling: Choose the behavior when a link is clicked.
+        By default FreeTube will open the link in your default browser.
+        (Unless it points to a Youtube Video/Playlist/Channel etc.)
+        You can choose to disable it completely for safety.
```

Comment:
1. This needs to be under the Tooltips section and follow the same conventions as it does there.
2. The "unless" clause again seems difficult to translate the way it is now grammatically.
3. This explanation does not talk about the prompting option at all.

My suggestion:
Choose the default behavior for interacting with links to content that cannot be opened in FreeTube.

Reply:
Translation entry moved and content updated